Modelica/Fluid/Dissipation.mo

```diff
@@ -4417,9 +4417,9 @@ This record is used as <strong>input record</strong> for the pressure loss funct
         DP := 0.5*IN_var.zeta_TOT*
           Modelica.Fluid.Dissipation.Utilities.Functions.General.SmoothPower(
                 m_flow,
-                (IN_con.dp_smooth/(0.5*IN_var.zeta_TOT*IN_var.rho))^0.5*IN_var.rho
+                sqrt(IN_con.dp_smooth/(0.5*IN_var.zeta_TOT*IN_var.rho))*IN_var.rho
             *IN_con.A_cross,
-                2)/(IN_var.rho*(IN_con.A_cross)^2);
+                2)/(IN_var.rho*sqrt(IN_con.A_cross));
```

Comment:
Maybe I'm reading it wrong, but this seems to turn a `^2` into a `sqrt`?

Reply:
Yes, corrected now.